public/pages/cadastro/cadastro_pais.html

```diff
@@ -146,7 +146,7 @@ <h4 class="mb-3">Informações</h4>
   <!---INICIO RODAPÉ-->
   <footer class=" footer-container py-5 conteiner-fluid row">
     <div class="col-6 col-md">
-        <img width= "200px + 10vw" src="/public/assets/images/logo.png">
+        <img width= "200px + 10vw" src="./public/assets/images/logo.png">
```

Comment:
Sugestão de correção:

```suggestion
        <img class="img-fluid" src="../../assets/images/logo.png">
```
Adicionar a propriedade ``width`` pelo css.

Obs.: Avaliação pendente...